### PR TITLE
fix: Set ClamAV package names for Ubuntu dists

### PR DESCRIPTION
**Pull request by monoflo**
## Description

This commit modifies the "Determine ClamAV package names" task to include Ubuntu distributions, preventing an undefined variable error when run on such hosts.

## Error Message

```
TASK [cisagov.clamav : Install ClamAV packages] ********************************
fatal: [ubuntu]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'package_names' is undefined\n\nThe error appears to be in '/home/user/.ansible/roles/cisagov.clamav/tasks/main.yml': line 15, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Install ClamAV packages\n  ^ here\n"}
```

## Playbook Workaround

Without this fix, the following workaround in `playbook.yml` is necessary:

```yml
---
- hosts: ubuntu
  gather_facts: yes

  roles:
    - role: cisagov.clamav
      vars:
        package_names: [ clamav-daemon ]  # here
```

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
Thanks for the PR, @monoflo!  This is a nice addition!

I made a few commits to add Ubuntu to the role metadata and molecule tests.

**Comment by jsf9k** [comment]
The Travis CI check won't work because Travis isn't enabled on this forked repo.  Once we have two approvals I'll temporarily turn off that requirement for the repo in order to merge in these changes.

**Comment by jsf9k** [comment]
The other check is failing because the job is taking too long to run. I will fix this when I have some free time later today.

**Review by mcdonnnj — APPROVED** [review]
This is a nice addition for this role. Nice addition @monoflo and thanks for cleaning this up @jsf9k 👍

File: tasks/main.yml
Comment (felddy) [inline tasks/main.yml:67]: I know this is perturbing the build... there are a couple reasons I want this completed before the first boot.
 - The AV report generator will be stating these file to get their age.
 - Not all instances have internet access.  It would be nice if they had some signatures even if they are a few weeks old.

**Review by felddy — CHANGES_REQUESTED** [review]
Let's talk about the removal of the signature download (see comment).

File: tasks/main.yml
Comment (jsf9k) [reply tasks/main.yml:67]: OK, I will revert the commits that removed that stuff.

Comment (jsf9k) [reply tasks/main.yml:67]: See 1182af8, 13287eb, and 909c548.

Comment (felddy) [reply tasks/main.yml:67]: Thanks @jsf9k

**Review by felddy — APPROVED** [review]
Thanks for the contribution @monoflo.  We really appreciate the extra sets of eyes.

Looks good. 👍

**Comment by jsf9k** [comment]
I think I can get the build to pass by splitting it into multiple jobs - one for each platform.  I will be experimenting with that now.

**Review by dav3r — APPROVED** [review]
Good work all - looks good to me! 🦅

**Pull request merged**
